.github/workflows/ci.yml

```diff
@@ -68,18 +68,21 @@ jobs:
         shell: bash
         run: ./scripts/build.sh -r
       - name: Start prometheus
+        if: secrets.PROMETHEUS_ID != ''
         shell: bash
         run: bash -x ./scripts/run_prometheus.sh
         env:
           PROMETHEUS_ID: ${{ secrets.PROMETHEUS_ID }}
           PROMETHEUS_PASSWORD: ${{ secrets.PROMETHEUS_PASSWORD }}
       - name: Start promtail
+        if: secrets.PROMETHEUS_ID != ''
```

Comment:
Would it make more sense to check for the vars that are used by the script? I think that would be `LOKI_ID` here.

either way lgtm